src/main/resources/assets/reignofnether/lang/en_us.json

```diff
@@ -1,42 +1,677 @@
+    "abilities.reignofnether.roar.tooltip3": "and briefly slowing down all nearby enemies. ",
+    "abilities.reignofnether.evoker_fangs_circular": "Evoker Fangs (Circular)",
+    "abilities.reignofnether.evoker_fangs_circular.tooltip1": "\\uE006  %d  \\uE004  %ds  \\uE005  ",
+    "abilities.reignofnether.evoker_fangs_circular.tooltip2": "Have this evoker summon a circle of snapping",
+    "abilities.reignofnether.evoker_fangs_circular.tooltip3": "fangs around itself when attacking.",
+    "abilities.reignofnether.evoker_fangs_line": "Evoker Fangs (Line)",
+    "abilities.reignofnether.evoker_fangs_line.tooltip1": "\uE006  %d  \uE004  %ds  \uE005  ",
+    "abilities.reignofnether.evoker_fangs_line.tooltip2": "Have this evoker summon a long line of snapping",
+    "abilities.reignofnether.evoker_fangs_line.tooltip3": "fangs around the caster when attacking.",
+    "abilities.reignofnether.sonic_boom": "Sonic Boom",
+    "abilities.reignofnether.sonic_boom.error1": "Must target a unit or building.",
+    "abilities.reignofnether.sonic_boom.tooltip1": "\uE006  %d  \uE004  %ds  \uE005  ",
+    "abilities.reignofnether.sonic_boom.tooltip2": "After a short delay, fire a targeted wave of sound at the",
+    "abilities.reignofnether.sonic_boom.tooltip3": "target, dealing heavy damage and knocking it far away.",
+    "abilities.reignofnether.teleport": "Teleport",
+    "abilities.reignofnether.teleport.tooltip1": "\uE004  %ds  \uE005  ",
+    "abilities.reignofnether.teleport.tooltip2": "Immediately teleport towards a location",
+    "abilities.reignofnether.harming_potion": "Harming Potion",
+    "abilities.reignofnether.harming_potion.tooltip1": "\uE006  3  \uE004  %ds  \uE005  ",
+    "abilities.reignofnether.harming_potion.tooltip2": "Throw a potion that deals instant damage to units.",
+    "abilities.reignofnether.healing_potion": "Healing Potion",
+    "abilities.reignofnether.healing_potion.tooltip1": "\uE007  3  \uE004  %ds  \uE005  ",
+    "abilities.reignofnether.healing_potion.tooltip2": "Throw a potion that restores health to units.",
+    "abilities.reignofnether.lingering_harming_potion": "Lingering Harming Potion",
+    "abilities.reignofnether.lingering_harming_potion.tooltip1": "\uE006  3  \uE004  %ds  \uE005  ",
+    "abilities.reignofnether.lingering_harming_potion.tooltip2": "Throw a potion that leaves a cloud of deadly gas.",
+    "abilities.reignofnether.lingering_healing_potion": "Lingering Healing Potion",
+    "abilities.reignofnether.lingering_healing_potion.tooltip1": "\uE007  3  \uE004  %ds  \uE005  ",
+    "abilities.reignofnether.lingering_healing_potion.tooltip2": "Throw a potion that leaves a cloud of healing vapours.",
+    "abilities.reignofnether.shield_stance": "Shield Stance",
+    "abilities.reignofnether.shield_stance.tooltip1": "Raise or lower a shield - reducing projectile ",
+    "abilities.reignofnether.shield_stance.tooltip2": "damage taken by 67%% and movement speed by 50%%.",
```

Comment:
`"abilities.reignofnether.shield_stance.tooltip2": "damage taken by 67% and movement speed by 50%.",`